modules/ROOT/partials/configuration/uploadcare_filters.adoc

```diff
@@ -0,0 +1,126 @@
+== `uploadcare_filters`
+
+The `+uploadcare_filters+` option defines the list of image filters available when using the {pluginname} plugin. Each filter applies a stylized image effect consistent with the visual design of the content and can be configured with an amount value to control its intensity. Filters enhance images by applying consistent styling, improving overall quality, and boosting user engagement.
+
+
+Each filter is defined as an object with the following properties:
+[source,js]
+----
+{
+  name: 'filterName', // Required: name of the filter to apply
+  amount: number, // Optional: strength of the filter (default = 100)
+  label: 'string' // Optional: custom name to display in the UI
+}
+----
+
+* `name`: A predefined filter identifier (e.g., adaris, zevcen, galen).
+* `amount`: The intensity of the filter effect, ranging from -100 to 200:
+** **0**: No effect (image stays original).
+** **0** to **100**: Gradually increases filter strength.
+** **100**: Default intensity (as the filter is designed).
+** **>100**: Exaggerates the effect.
+** **<0**: Inverts the filter effect (produces a "negative" version).
+* `label`: Optional display name for the filter in your UI. If omitted, it defaults to the name.
+
+=== Example: Setting `uploadcare_filters`
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",
+  plugins: 'uploadcare',
+  toolbar: 'uploadcare',
+  uploadcare_public_key: '<your-public-key>',
+
+  // Define available Uploadcare image filters and their intensity levels
+  uploadcare_filters: [
+    { name: 'none' }, // No filter applied
+    { name: 'adaris', amount: -100 }, // Adaris with inverted effect (amount -100), label defaults to 'adaris'
+    { name: 'adaris', amount: -100, label: 'Vintage Fade' }, // Adaris with inverted effect (amount -100), label reads 'Vintage Fade'
+    { name: 'adaris', amount: 0, label: 'Base' }, // Adaris with neutral effect (amount 0), label reads 'Base'
+    { name: 'adaris', amount: 50, label: 'Light' }, // Adaris with light effect (amount 50), label reads 'Light'
+    { name: 'adaris', amount: 100, label: 'Standard' }, // Adaris with standard effect (amount 100), label reads 'Standard'
+    { name: 'adaris', amount: 200, label: 'Intense' }, // Adaris with intense effect (amount 200), label reads 'Intense'
+    { name: 'zevcen', amount: 200, label: 'Glow Boost' }, // Zevcen with intense effect (amount 200), label reads 'Glow Boost'
+    { name: 'galen', amount: 80, label: 'Soft Focus' }, // Galen with softening effect (amount 80), label reads 'Soft Focus'
+    { name: 'carris', amount: 120, label: 'Sharp Contrast' }, // Carris with high contrast (amount 120), label reads 'Sharp Contrast'
+    { name: 'ferand', amount: 60, label: 'Light Touch' }, // Ferand with light enhancement (amount 60), label reads 'Light Touch'
+    { name: 'sorahel', amount: -50, label: 'Night Mood' } // Sorahel with darkened effect (amount -50), label reads 'Night Mood'
+  ]
+});
+----
+
+[NOTE]
```

Comment:
@kemister85 I expect NOTE to be like this

![image](https://github.com/user-attachments/assets/6b692e00-fbb1-4613-a3f6-b17629a21ea9)

Reply:
Resolved in last [commit](https://github.com/tinymce/tinymce-docs/pull/3679/commits/0bf4b9afdcb2fa031f6708e63d5a0adf12657ec2)